tables/lg-ug-g1.utb

```diff
@@ -0,0 +1,58 @@
+# liblouis: Luganda Grade 1 Braille
+#
+#-display-name: Luganda Grade 1
+#-index-name: Luganda Grade 1
+#
+#-author-name: Jake Kyle
+#-maintainer-name: Jake Kyle
+#-maintainer-email: jake@compassbraille.org
+#
+#+language: lg
+#+region: ug
+#+type: literary
+#+contraction: no
+#+grade: 1
```

Comment:
Do other grades exist?

Reply:
No other grades exist as far as I know. Shall I remove:
`#+grade: 1`?

Reply:
Perhaps that's best, yes.